wdfn-server/waterdata/templates/monitoring_location.html

```diff
@@ -85,14 +85,16 @@ <h1>{{ stations[0].station_nm }}</h1>
                             var splitUrl = urlString.split('#')
                             var siteNumber = splitUrl[0].split('/monitoring-location/')[1].slice(0, -1);
                             var paramString = splitUrl[1];
+
+                            if (typeof(paramString) === "undefined") {
+                                paramString = 'parameterCode=00060'
```

Comment:
The default parameter is not always 00060, specifically for sites that don't have Discharge. You should be able to get the "default parameter" by looking at the template variable parm-grp_summary: https://github.com/usgs/waterdataui/blob/master/wdfn-server/waterdata/views.py#L126